README.md

```
@@ -103,19 +100,13 @@ Instalocking is a bannable offense. You likely won't get banned unless you go ar
 
-- You can hit me up on Discord [@lternatively](https://discord.com/users/735322421862727760)
-
-- If your issue is "cannot initialize a valclient", follow this method to report it https://github.com/SuppliedOrange/VALORANT-Instalocker/issues/12
-
-- For other issues, use the `--debug` parameter to log everything that happens when you try to instalock then make an issue:
+- For issues, use the `--debug` parameter to log everything that happens when you try to instalock then open a GitHub issue:
   [Method 1](https://i.imgur.com/InEbwdz.mp4) [Method 2](https://i.imgur.com/R5ElrrP.png) <br>
   
 - Sometimes the instalocker waits for a few seconds after the loading screen to instalock, this is because it scans for the pre-game screen every 3 seconds. You can change the loop delay in the code for yourself.
 
 #### Notes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)